github/review/review.rb

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env ruby
+
+# This script is intended for mass review of same (or very similar)
+# GitHub pull requests.
+#
+# Usage:
+# $PROGRAM_NAME [--merge] [--delete] <PR_URL> <PR_URL> ...
+#   -m --merge     - merge the approved pull requests
+#   -d --delete    - delete the source branch after merging
+#                  (skipped when the PR is opened from a fork)
+#
+# This script displays the details about the specified pull requests
+# (the title, diff, Travis status).
+#
+# If everything is fine you can approve the pull request and
+# optionally merge it and remove the source branch by entering
+# "y" or just pressing [Enter] when it asks for approval.
+#
+# It can only approve the pull requests, if you do not want to
+# approve a pull request you need to skip it and comment manually
+# at the GitHub pull request web page.
+#
+
+require "shellwords"
+
+# install missing gems
+vendor_dir = File.join(__dir__, ".vendor")
+bundle_dir = File.join(vendor_dir, "bundle")
+
+ENV["BUNDLE_GEMFILE"] = File.join(__dir__, "Gemfile")
+
+if !File.exist?(vendor_dir)
+  puts "Installing the needed Rubygems to #{bundle_dir} ..."
+  system "bundle install --path #{bundle_dir.shellescape}"
+end
+
+require "rubygems"
+require "bundler/setup"
+
+require "octokit"
+require "rainbow"
+require "optparse"
+
+# global options
+merge_pr = false
+delete_branch = false
+
+# parse the command line options
+OptionParser.new do |parser|
+  parser.banner = "Usage: #{$PROGRAM_NAME} [options] PR_URL1 PR_URL2 ..."
```

Comment:
I would write here also how to setup authentication. banner allows 'n, so probable on next line.